.gitlab/package_build/deb.yml

```diff
@@ -19,6 +19,10 @@
     - chmod 0744 /tmp/system-probe/clang-bpf /tmp/system-probe/llc-bpf
     - inv -e omnibus.build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod" --system-probe-bin=/tmp/system-probe ${FLAVOR:+--flavor $FLAVOR}
     - ls -la $OMNIBUS_PACKAGE_DIR
+    - curl -sSL "https://dd-package-tools.s3.amazonaws.com/dd-pkg/${DD_PKG_VERSION}/dd-pkg_Linux_${DD_PKG_ARCH}.tar.gz" | tar -xz -C /usr/local/bin dd-pkg
+    - pushd $OMNIBUS_PACKAGE_DIR
+    - find . -iregex '.*\.\(deb\|rpm\)' -printf '%f\0' | xargs -0 dd-pkg lint
```

Comment:
wouldn't it be equivalent to just provide `$OMNIBUS_PACKAGE_DIR` to `find`?

Reply:
You'd also need to prefix the results with `$OMNIBUS_PACKAGE_DIR` into the `xargs` invocation, which is also doable - happy to look at that if we'd prefer that.

Reply:
Not necessarily, find will prefix the results if the path isn't contained in the current working directory I think:
```
 ⚙  ~/dd/cloud-inventory   chouquette/allow_getbucketversioning  pwd
/home/hugo.beauzee/dd/cloud-inventory
 ⚙  ~/dd/cloud-inventory   chouquette/allow_getbucketversioning  find ~/dd/datadog-agent -name agent.py
/home/hugo.beauzee/dd/datadog-agent/tasks/agent.py
```

Reply:
Oh no my bad, it just prefixes with whatever folder you're searching into

Reply:
Which, I guess should work? I swear it just passed in the filename without the path when I was testing it 🤔

Reply:
I remembered what happened here, misremembered there was another reason it wasn't working (not the `find` command). I'll update this for simplicity. Thanks!

Reply:
Maybe I was wrong about my memory being wrong, I'm getting [this error](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/496947875#L1217) where it's returning the found files with the relative path and then failing to find them for linting.

Reply:
Does it work if you remove the `-printf %F\0`? `%F` will only display the basename which I suppose is why the path isn't correct

Reply:
:duh: thanks!